example.nxc

```diff
@@ -6,18 +6,41 @@
 task main()
 {
     SetSensorNXTViSy(PORT);
-    NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    long err = NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    if (err != NO_ERR) {
+        TextOut(0, LCD_LINE4, "NXTViSyInit error: ");
+        NumOut(90, LCD_LINE4, err);
+        while(1) {
+            ;
+        }
+    }
 
     int distance;
     byte action;
     string writable_action = "x";
 
     while (true) {
-        NXTViSyGetData(distance, action);
+        ClearScreen();
+        err = NXTViSyGetData(distance, action);
+        if (err == GET_DATA_I2C_BYTES_ERR) {
+            TextOut(0, LCD_LINE5, "I2CBytes error in function");
```

Comment:
@vargac could you please refactor these 5 lines into a function that would take two arguments and do precisely what these 5 lines do (that is print out those two lines and wait forever).